cli.py

```diff
@@ -67,21 +75,62 @@ def main(log_level):
 @click.option('-d', '--domain-name', default=DEFAULT_DOMAIN_NAME,
               type=str, help="D3A domain name")
 @click.option('-w', '--web-socket', default=DEFAULT_WEBSOCKET_DOMAIN,
-              type=str, help="D3A websocket URL")
-@click.option('--run-on-redis', is_flag=True, default=False, help="Start the client using the Redis API")
-def run(setup_module_name, username, password, domain_name, web_socket, run_on_redis, **kwargs):
+              type=str, help="D3A websocket URI")
+@click.option('-i', '--simulation-config-path', type=str,
+              help="Path to simulation config file (accept absolute and relative path)")
+@click.option('--run-on-redis', is_flag=True, default=False,
+              help="Start the client using the Redis API")
+def run(base_setup_path, setup_module_name, username, password, domain_name, web_socket,
+        simulation_config_path, run_on_redis, **kwargs):
     if username is not None:
         os.environ["API_CLIENT_USERNAME"] = username
     if password is not None:
         os.environ["API_CLIENT_PASSWORD"] = password
     os.environ["API_CLIENT_DOMAIN_NAME"] = domain_name
     os.environ["API_CLIENT_WEBSOCKET_DOMAIN_NAME"] = web_socket
     os.environ["API_CLIENT_RUN_ON_REDIS"] = "true" if run_on_redis else "false"
+    set_simulation_file_env(base_setup_path, simulation_config_path, run_on_redis)
+
+    load_client_script(base_setup_path, setup_module_name)
+
 
+def load_client_script(base_setup_path, setup_module_name):
     try:
-        importlib.import_module(f"d3a_api_client.setups.{setup_module_name}")
+        if base_setup_path is None:
+            importlib.import_module(f"d3a_api_client.setups.{setup_module_name}")
+        elif base_setup_path and os.path.isabs(base_setup_path):
+            sys.path.append(base_setup_path)
+            importlib.import_module(setup_module_name)
+        else:
+            setup_file_path = os.path.join(os.getcwd(), base_setup_path)
+            sys.path.append(setup_file_path)
+            importlib.import_module(setup_module_name)
 
     except D3AException as ex:
         raise click.BadOptionUsage(ex.args[0])
     except ModuleNotFoundError as ex:
         log.error("Could not find the specified module")
+
+
+def set_simulation_file_env(base_setup_path, simulation_config_path, run_on_redis):
+    if run_on_redis is True:
+        os.environ["SIMULATION_CONFIG_FILE_PATH"] = ""
+        return
+    if simulation_config_path is None:
```

Comment:
Are we testing for `None` values or, more generally, for false truthiness values? This version would cover both:
```suggestion
    if not simulation_config_path:
```